front/src/Components/DrawingList/index.tsx

```diff
@@ -16,33 +24,73 @@ const parseDate = (date: string) => (
 );
 
 export const DrawingList = ({ files, onFileClick }: DrawingListProps) => {
+  const [searchQuery, setSearchQuery] = useState('');
 
-  const sortedFiles = [...files];
-  sortedFiles.sort((a, b) => {
-    const tsA = dateAsTimestamp(a.created_at);
-    const tsB = dateAsTimestamp(b.created_at);
-    return tsB - tsA;
+  const [sortedFiles, setSortedFiles] = useState<DrawingMetadata[]>([]);
+
+  useEffect(() => {
+    let realFiles;
```

Comment:
you forgot to type this... baka!
![bakaaa](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExaGM4MXY2YmVvZnllYWVic25raHB6YTdjNnByM2c2ZDUwMDI4ajU3ayZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3uv5ivx6xwqze/giphy.gif)

Reply:
[I don't know if you should really play this game](https://github.com/search?q=repo%3APatitoDev%2FElPatoDraw+%2Fany%2F+language%3ATSX&type=code).

However, [fixed](https://github.com/PatitoDev/ElPatoDraw/pull/4/commits/840b5fa8b71b8247db8610bce513d8a49195b91d).

Reply:
this is necessary to how weird the excalidraw data structure is set as, and *should* be the only exception. The only reason why I have not disallowed any on the tsconfig so this was a conscious decision

Reply:
mainly because excalidraw does not expose their types correctly :/

Reply:
Okay, it was a joke, don't get so defensive on the anies.

Accept this GIF of a human petting a duck.

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExejd3YXBmajU3bTI0enh3dXlwZnIxNXIwcmtpcnhpNmNndHQyc3lmdSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/wsUtUtLR3A2XPvfLVs/giphy-downsized.gif)

Reply:
:v